src/test/java/org/jenkinsci/plugins/envinject/BuildCauseRetrieverTest.java

```diff
@@ -1,121 +0,0 @@
-package org.jenkinsci.plugins.envinject;
```

Comment:
I've completely rewrite tests in this class. This is the main cause why git not handles this move. Tests save the logic but became simpler. And understandable - just read them.